cmd/kubeadm/app/master/apiclient.go

```diff
@@ -223,6 +220,18 @@ func SetMasterNodeAffinity(meta *api.ObjectMeta) {
 	meta.Annotations[api.AffinityAnnotationKey] = string(affinityAnnotation)
 }
 
+func MasterNodeAffinity() api.NodeSelectorRequirement {
+	return api.NodeSelectorRequirement{
+		Key: "kubeadm.alpha.kubernetes.io/role", Operator: api.NodeSelectorOpIn, Values: []string{"master"},
+	}
+}
+
+func MonoArchitectureNodeAffinity() api.NodeSelectorRequirement {
```

Comment:
The term "Mono" here makes me think of [Mono](http://www.mono-project.com/) and windows and such.  Perhaps just "ArchitectureNodeAffinity"?  If we add multi-arch support in the future we can rename/rework this.